src/registrar/tests/test_views.py

```diff
@@ -1197,6 +1198,13 @@ def test_domain_invitation_created(self):
         EMAIL = "mayor@igorville.gov"
         User.objects.filter(email=EMAIL).delete()
 
+        # Create an application
```

Comment:
You may still need to create an application and approve it, to get the side effects of domain and domainInfo creation (see my comment on using DI instead of DA in the view file)